devices/tuya/_TZE200_3towulqd_motion_lux_sensor.json

```diff
@@ -0,0 +1,153 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "_TZE200_3towulqd",
+  "modelid": "TS0601",
+  "vendor": "Tuya",
+  "product": "ZG-204ZL",
```

Comment:
Can you add which type of device this is, like xyz presence sensor?

Reply:
done